build_and_push.sh

```diff
@@ -15,13 +15,15 @@ function echo_run() {
     "$@"
 }
 
-# default REMOTE
+# default argument values
 REMOTE="docker.io/finkandreas"
+CONFIG_FILE="build-versions.yaml"
 
-while getopts "lhr:" Option
+while getopts "lhrf:" Option
```

Comment:
this should be `lhr:f:`, because both `-r` and `-f` require an additional argument.

Reply:
Stupid mistake